src/main/java/main/internet/shop/security/filter/AuthorisationFilter.java

```diff
@@ -0,0 +1,83 @@
+package main.internet.shop.security.filter;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import main.internet.shop.lib.Injector;
+import main.internet.shop.model.User;
+import main.internet.shop.model.role.UserRole;
+import main.internet.shop.service.UserService;
+
+public class AuthorisationFilter implements Filter {
+    private static final String USER_ID = "userId";
+    private static final Injector injector =
+            Injector.getInstance("main.internet.shop");
+    private Map<String, List<UserRole>> protectedUrls;
+    private UserService userService = (UserService)
+            injector.getInstance(UserService.class);
+
+    @Override
+    public void init(FilterConfig config) throws ServletException {
```

Comment:
would be easier to check if you separate ADMIN pages from USER pages
it's hard to read when it's mixed